### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by mrogozhinskaya**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Hello world
 
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_hello-world/)
```
Comment (mrogozhinskaya) [inline readme.md:4]: don`t use this task

Comment (mrogozhinskaya) [reply readme.md:4]: - [DEMO LINK](https://mrogozhinskaya.github.io/layout_hello-world/)
- [TEST REPORT LINK](https://mrogozhinskaya.github.io/layout_hello-world/report/html_report/)

**Review by OleziO — APPROVED** [review]

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+  </head>
+  <body>
+   Hello, world!
```
Comment (OleziO) [inline src/index.html:10]: Wrap with some text tag.

**Review by OleziO — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 2 above)
Comment (mrogozhinskaya) [reply src/index.html]: Ready!
Check, please!

**Review by OleziO — APPROVED** [review]

**Pull request closed without merging**